Removes Course org buttons on about page

kabirkhan opened the pull request.

Commit:
Adding .gitignore for compiled css. Deleting course organization button from course about page for both sites.

Commit:
Merge branch 'oxa/master.fic' into oxa/develop.fic

kabirkhan commented:
Don't merge yet.

ms-hukoyun reviewed (changes requested):
For .gitignore folder name must be courses not mytraining

Commit:
Replaced mytraining with courses in a few files to get correct asset paths. Fixed bug so that enrolling in ReX course redirects to course info page instead of dashboard.

Commit:
Update _overrides.scss. 

Removes unnecessary change.

Commit:
Update _overrides.scss

Same thing

kabirkhan commented:
Fixed. 
Also changed all references to "mytraining" to "courses" to correctly pick up static assets.
Fixed bug that redirected user to dashboard. Now redirects to course home page.
Removed org buttons from course about page.
